client/src/pages/ElementPage.css

```diff
@@ -0,0 +1,51 @@
+.element-page {
```

Comment:
CSS files are big headaches! If you really want to do style, you should use "sx" or "style" property.